src/composable/composable.ts

```diff
@@ -166,6 +166,40 @@ function sequence<T extends [Composable, ...Composable[]]>(
   >
 }
 
+/**
+ * Sequences a tuple of functions and collect the output of every function like `sequence` feeding the same input to all of them.
+ * Will abort the sequence on the first error.
+ * @example
+ * import { composable as C } from 'domain-functions'
+ *
+ * const a = C.composable((something: unknown) => String(something))
```

Comment:
Maybe change the output type to better show the input of the next function is not related to it

```suggestion
 * const a = C.composable((something: unknown) => Number(something))
```